lib/chef/mixin/user_context.rb

```diff
@@ -0,0 +1,52 @@
+#
+# Author:: Adam Edwards (<adamed@chef.io>)
```

Comment:
[minor] Is this author line correct?  Also the copyright line below can say 2017.